packages/mux-player/src/index.ts

```diff
@@ -15,34 +15,122 @@ import { toNumberOrUndefined } from "./utils";
 import type { MuxTemplateProps } from "./types";
 import type { Metadata } from "@mux-elements/playback-core";
 
-class MuxPlayerInternal {
-  protected el: MuxPlayerElement;
-  protected _resizeObserver?: ResizeObserver;
-  protected _state: Partial<MuxTemplateProps> = {
+export type Tokens = {
+  playback?: string;
+  thumbnail?: string;
+  storyboard?: string;
+};
+
+const SMALL_BREAKPOINT = 700;
+const XSMALL_BREAKPOINT = 300;
+const MediaChromeSizes = {
+  LG: "large",
+  SM: "small",
+  XS: "extra-small",
+};
+
+function getPlayerSize(el: Element) {
+  const muxPlayerRect = el.getBoundingClientRect();
+  return muxPlayerRect.width < XSMALL_BREAKPOINT
+    ? MediaChromeSizes.XS
+    : muxPlayerRect.width < SMALL_BREAKPOINT
+    ? MediaChromeSizes.SM
+    : MediaChromeSizes.LG;
+}
+
+const MuxVideoAttributes = {
```

Comment:
Should these (and `PlayerAttributes`, `getProps` and the Names objects) move into some kind of `props.ts` file?

Reply:
could be nice. we can revisit this.